pyproject.toml

```diff
@@ -59,20 +59,20 @@ dependencies = [
   "ipykernel >= 5.5.6",
   "ipywidgets >= 6.0.0",
   "lmfit >= 1.1.0",
-  "matplotlib >= 3.5.1",
+  "matplotlib >= 3.6.0",
   "mpmath >= 1.2.1",
   "numba >= 0.57.0",
-  "numpy >= 1.23.0, < 2.0.0",
+  "numpy >= 1.24.0",
   "packaging >= 22.0",
-  "pandas >= 1.4.0",
+  "pandas >= 1.5.0",
   "requests >= 2.28.0",
-  "scipy >= 1.8.0",
+  "scipy >= 1.9.0",
   "setuptools >= 61.2.0",
-  "tqdm >= 4.60.0",
+  "tqdm >= 4.65.0",
   # Use loose requirements for voila because it impacts ipykernel version
   "voila >= 0.2.7",
   "wrapt >= 1.14.0",
-  "xarray >= 2022.3.0",
+  "xarray >= 2022.6.0",
```

Comment:
These updates are all based on the SPEC 0 recommended policy (i.e., that we drop support for dependencies that were released ≥ 24 months ago).